v1/README.md

```diff
@@ -36,6 +36,7 @@ Class | Method | HTTP request | Description
 *RadiusServersApi* | [**RadiusServersList**](docs/RadiusServersApi.md#radiusserverslist) | **Get** /radiusservers | List Radius Servers
 *RadiusServersApi* | [**RadiusServersPost**](docs/RadiusServersApi.md#radiusserverspost) | **Post** /radiusservers | Create a Radius Server
 *RadiusServersApi* | [**RadiusServersPut**](docs/RadiusServersApi.md#radiusserversput) | **Put** /radiusservers:id | Update Radius Servers
+*SearchApi* | [**SearchOrginzationsPost**](docs/SearchApi.md#searchorginzationspost) | **Post** /search/organizations | Search Organizations
```

Comment:
`searchorganizationspost`? `SearchOrganizationsPost`?

Reply:
I'll fix this in another PR, it's just the slug / operator ID which is a stoplight docs thing.